docs/src/pages/ui/primitives/icon/react.mdx

```diff
@@ -85,29 +86,54 @@ const ThumbsUpIcon = () => {
 <Example>
   <Icon
     ariaLabel="Thumbs up"
-    pathData="M9 21h9c.83 0 1.54-.5 
-  1.84-1.22l3.02-7.05c.09-.23.14-.47.14-.73v-2c0-1.1-.9-2-2-2h-6.31l.95-4.57.03-.32c0-.41-.17-.79-.44-1.06L14.17 
+    pathData="M9 21h9c.83 0 1.54-.5
+  1.84-1.22l3.02-7.05c.09-.23.14-.47.14-.73v-2c0-1.1-.9-2-2-2h-6.31l.95-4.57.03-.32c0-.41-.17-.79-.44-1.06L14.17
   1 7.58 7.59C7.22 7.95 7 8.45 7 9v10c0 1.1.9 2 2 2zM9 9l4.34-4.34L12 10h9v2l-3 7H9V9zM1 9h4v12H1z"
   />
 </Example>
 
 ### Sizes
 
-Use the `size` prop to change the Icon size. Available options are `small`, `large`, and none (default).
+Icon size matches the font-size of the container. Adjust the font-size to set a specific height.
 
-```jsx
-import { IconAdd } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
+_Inherited size from button sizes_
 
-<IconAdd size="small" />;
-<IconAdd />;
-<IconAdd size="large" />;
+```jsx
+  <Button gap="0.1rem" size="small" >
+    <IconSave /> Save
+  </Button>
+  <Button gap="0.2rem">
+    <IconSave /> Save
+  </Button>
+  <Button  gap="0.2rem" size="large">
+    <IconSave /> Save
+  </Button>
 ```
 
 <Example>
-  <IconAdd size="small" />
+  <Button gap="0.1rem" size="small">
+    <IconSave /> {'Save'}
```

Comment:
Do we in `{'Save'}` could it just be `Save`?

Reply:
Unfortunately MDX will convert `Save` to `<p>Save</p>` which breaks the styling and is invalid HTML.